lib/dw/dataset/column.js

```diff
@@ -225,6 +225,15 @@ export default function Column(name, rows, type) {
             return column;
         },
 
+        deleteRow(i) {
+            if (!arguments.length) return;
```

Comment:
I would remove this condition and just let the method crash if it's called without arguments (which is wrong and unsupported).

Reply:
how about
```suggestion
            if (!arguments.length) throw new Error('deleteRow must be called with a row index');
```
then?

Reply:
You could add such a condition to any function that takes arguments. Like `Column.each()`, `Column.limitRows()`, `Dataset.deleteRow()` etc. I really think crashing is fine. I wouldn't be super defensive.